config/configuration.md

```diff
@@ -201,6 +201,19 @@ The global coefficient applied to all deviation checks. Used to differentiate al
 
 Defaults to `1`.
 
+#### `asyncBeaconUpdatedeviationThresholdFactor` _(optional)_
+
+The global deviation factor applied to async beacon updates deviation checks. When configured, this value will be used
+on beacons sets that don't require an update but some of its beacon constituents do. So if a beacon set has a 1%
+deviation threshold, Airseeker will only update its beacons when deviation of each beacon exeeds (1 \*
+asyncBeaconUpdatedeviationThresholdFactor)%. For example:
```

Comment:
The text should also mention:
> - This (allowing asynchronous updates of Beacons) is an intentional tradeoff. We do some redundant Beacon updates in return for resiliency against unavailability.
> - It is not unheard of for a single Beacon to spaz out (e.g. JPY/USD that flips between Yen and new Yen conventions). Therefore, it's possible for this to drain the wallet, which means this should only be enabled on the main or the fallback (fixed sponsor wallet) Airseeker configurations. So whether this is enabled should be config-configurable.

(in a shortened way though).

Reply:
https://github.com/api3dao/airseeker/pull/343/commits/e1183d04a3651a1af27d8726e7812fa106a6b2bc